src/katello/enabled_report.py

```diff
@@ -0,0 +1,33 @@
+class EnabledReport(object):
+    """
+    Represents the enabled repos report.
+    @ivar content: The report content <dict>:
+        - repos[] <dict>:
+        - repositoryid <str>
+        - baseurl <str>
+    :type content: dict
+    """
+    def __find_enabled(self):
+        """
+        Get enabled repos part of the report.
+        :return: The repo list content
+        :rtype: dict
+        """
+        enabled =  [dict(repositoryid=name, baseurl=self.repofile.get(name, "baseurl")) \
```

Comment:
I got a lot of feedback on this in my last pr - use {} instead of dict() as it's more pythonic

Reply:
And does this line work on python2.4? If we are going to backport usage of this approach we need to be sure

Reply:
Would also like some tests for this class

Reply:
> And does this line work on python2.4? If we are going to backport usage of this approach we need to be sure

Yup verified on RHEL 5

Reply:
> I got a lot of feedback on this in my last pr - use {} instead of dict() as it's more pythonic

Interesting I 'd argue the latter :). dict is more pythonic, but a good idea to stick to a standard. I'll move to your way.